azureproject/settings.py

```diff
@@ -93,3 +93,67 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+if 'RUNNING_IN_PRODUCTION' in os.environ:
+    # Configure allowed host names that can be served and trusted origins for Azure Container Apps.
+    ALLOWED_HOSTS = ['.azurecontainerapps.io']
+    CSRF_TRUSTED_ORIGINS = ['https://*.azurecontainerapps.io']
+    DEBUG = False
+    DEBUG_PROPAGATE_EXCEPTIONS = True
+
+    # SECURITY WARNING: keep the secret key used in production secret!
+    # Use this py command to create secret 
+    # python -c 'import secrets; print(secrets.token_hex())'
+    SECRET_KEY = os.getenv('AZURE_SECRET_KEY')
+
+    STATICFILES_STORAGE = 'whitenoise.storage.CompressedStaticFilesStorage'
```

Comment:
I think I now use whitenoise locally as well in my django apps for consistency, but I do not feel strongly about it.